packages/gasket-plugin-command/README.md

```diff
@@ -24,212 +25,220 @@ export default makeGasket({
 });
 ```
 
+---
+
 ## Lifecycles
 
-### Commands
+### `commands`
```

Comment:
Too keep consistent with our [markdown guidelines](https://github.com/godaddy/gasket/blob/main/CONTRIBUTING.md#markdown-documentation), avoid inline code in headings:
```suggestion
### commands
```